Spectacle/Resources/Localizations/fr.lproj/Spectacle.strings

```diff
@@ -88,6 +88,11 @@
 
 /* --- */
 
+/* Class = "NSMenuItem"; title = "Restore Defaults..."; ObjectID = "aFM-wq-kOy"; */
+"aFM-wq-kOy.title" = "Rétablir par défaut...";
```

Comment:
I don't think this is necessary. This menu item has been removed in favor of the button in the preferences window.